kafka/admin/client.py

```diff
@@ -107,6 +107,8 @@ class KafkaAdminClient(object):
         ssl_check_hostname (bool): Flag to configure whether SSL handshake
             should verify that the certificate matches the broker's hostname.
             Default: True.
+        ssl_cadata (str): optional ca filecontent to use in certificate
+            verification. default: None.
```

Comment:
Can you capitalize properly?

```suggestion
        ssl_cadata (str): Optional CA filecontent to use in certificate
            verification. Default: None.
```

This applies several other places in the PR too, since this text is sorta copy/pasta.